### PR TITLE
Quiz component

### PR DESCRIPTION
**Pull request by mhmdtshref**

### CODE REVIEW

**Review by ameentalahmeh — APPROVED** [review]
seen :+1:

**Review by RazanTayyem — APPROVED** [review]
Good  :+1:

File: student/src/App.js [hunk 1]
```diff
@@ -4,20 +4,18 @@ import {Route, BrowserRouter as Router} from 'react-router-dom';
...
-
-
+import Quiz from './components/Quiz';
 
 class App extends Component {
   render() {
   return (
     <Router>
-    <div>
+    <div id="root">
```
Comment (ghassanmas) [inline student/src/App.js:14]: This `id="root"` is not needed here

File: student/src/components/Quiz/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        this.getQuestionsIDs(this.props.quizId).then((questionsIds) => {
+            this.setState({ questionsIds });
+        }).catch(() => {
+            alert("Error getting questions!")
+        })
+    }
+
+    onClickNext = () => {
+        this.setState((state) => ({questionCounter: ++state.questionCounter}) );
+    };
```
Comment (ghassanmas) [inline student/src/components/Quiz/index.js:35]: to differentiate call it `prevState` instaed of `state`

**Review by ghassanmas — APPROVED** [review]

**Pull request merged**